src/components/general/modals/share-experiences-modal.js

```diff
@@ -84,8 +84,8 @@ export default function ShareExperiencesModal(props) {
       .catch((err) => {
         setExperienceResult({
           title: 'Please try again tomorrow.', 
-          content: `Experiences can only be shared once per day. You left a review of `+props.testTitle+` less than 24 hours ago.
-                    Please wait 24 hours from the time you last shared an experience before sharing another. Thank you!`
+          content: `To help ensure authenticity, you'll need to wait until tomorrow
```

Comment:
since youre using back ticks you can use template literals the new way with ${} syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thanks, good catch. Just changed. Definitely looks neater